Description: Do not use SSLv23 by default, but TLSv1
 Default protocol should really be TLSv1 and not SSLv23.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2015-05-21

--- a/eventlet/green/ssl.py
+++ b/eventlet/green/ssl.py
@@ -49,7 +49,7 @@
 
     def __init__(self, sock, keyfile=None, certfile=None,
                  server_side=False, cert_reqs=CERT_NONE,
-                 ssl_version=PROTOCOL_SSLv23, ca_certs=None,
+                 ssl_version=PROTOCOL_TLSv1, ca_certs=None,
                  do_handshake_on_connect=True, *args, **kw):
         if not isinstance(sock, GreenSocket):
             sock = GreenSocket(sock)
